docs/commands/playsound.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Playsound
+category: Commands
+mention:
+    - BedrockCommands
+    - zheaEvyline
+tags:
+    - info
+---
+
+## Introduction
+
+[Sourced By Bedrock Commands Community Discord](https://discord.gg/SYstTYx5G5)
+
+Introduced in version `1.0.5`, you can use the `/playsound` command to play sound effects to players present anywhere in the world whenever you like.
```

Comment:
If it's old, we shouldn't need an introduce in. Should only be valid for 1.16 or higher